pkg/api/container_repository_api.go

```diff
@@ -0,0 +1,58 @@
+package api
+
+import (
+	"context"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+)
+
+// CreateContainerRepositoryOpts contains necesessary information to create a container repository
+type CreateContainerRepositoryOpts struct {
+	ClusterID ID
```

Comment:
We call this `ID` everywhere else, super minor detail, I know, but might as well stick with it

Reply:
Discussed in DM and agreed upon renaming the api.ID struct to api.ClusterID